aurora-solidity-sdk/src/Types.sol

```diff
@@ -0,0 +1,66 @@
+// SPDX-License-Identifier: CC-BY-1.0
+pragma solidity ^0.8.17;
+
+/// Basic NEAR promise.
+struct PromiseCreateArgs {
+    /// Account id of the target contract to be called.
+    string targetAccountId;
+    /// Method in the contract to be called
+    string method;
+    /// Payload to be passed to the method as input.
+    bytes args;
+    /// Amount of NEAR tokens to attach to the call. This will
+    /// be charged from the caller in wNEAR.
+    uint128 nearBalance;
+    /// Amount of gas to attach to the call.
+    uint64 nearGas;
+}
+
+enum PromiseArgsVariant
+/// Basic NEAR promise
```

Comment:
`forge fmt` (the formatter that comes with the `forge` tool) puts the comments in this position. I agree it looks strange, but I will leave it this way since I don't want to have to remember to correct this every time after running the formatter.